src/cmd/ramfs.c

```diff
@@ -875,7 +859,7 @@ void *
 erealloc(void *p, ulong n)
 {
 	p = realloc(p, n);
-	if(!p)
+	if(!p && n != 0)
```

Comment:
So this is a little bit weird, but `realloc` is being changed in C23 and in general, to avoid future complications, I think it's probably best to follow the new standard behavior (even though plan9 realloc is different etc etc).  I'd probably rewrite this as,

    void *
    erealloc(void *p, ulong n)
    {
        if(n == 0){
            free(p);
            return nil;
        }
        p = realloc(p, n);
        if(!p)
            error("out of memory");
        return p;
    }